src/content/en/tools/chrome-devtools/remote-debugging/index.md

```diff
@@ -156,3 +157,53 @@ You can interact with the screencast in multiple ways:
 * To scroll, use your trackpad or mouse wheel, or fling with your mouse
   pointer.
 
+Some notes on screencasts:
+
+* Screencasts only display page content. Transparent portions of the screencast 
+  represent device interfaces, such as the Chrome omnibox, the Android status 
+  bar, or the Android keyboard. 
+* Screencasting continuously capture frames, so disable it if your testing
```

Comment:
It makes sense to me, but I can see Joe's point. Maybe phrase as "Screencasting continually captures frames, which may affect the performance of your app." 

In any event, your/you're ;)